distribution/docs/src/main/resources/content/_architectures/url-resource-reader.adoc

```diff
@@ -31,6 +31,25 @@ Configure the URL Resource Reader from the ${admin-console}.
 
 See <<ddf.catalog.resource.impl.URLResourceReader,URL Resource Reader configurations>> for all possible configurations.
 
+=== Configuring Permissions for the URL Resource Reader
+
+Configuring the URL Resource Reader to retrieve files requires adding permissions to the Security Manager.
```

Comment:
❓ Do we need this sentence here? Can we combine with the first sentence in the next paragraph?

Reply:
combining into one sentence